scapy/sendrecv.py

```diff
@@ -88,7 +88,9 @@ def _sndrcv_rcv(pks, tobesent, stopevent, nbrecv, notans, verbose, chainCC,
         h = i.hashret()
         hsent.setdefault(i.hashret(), []).append(i)
 
-    if WINDOWS:
+    from scapy.contrib.cansocket import is_python_can_socket
```

Comment:
Could you put the imports at the top of the file?

Reply:
I changed my code a little bit, to avoid loading my contrib module inside the Scapy core function